crates/bevy_ui/src/lib.rs

```diff
@@ -75,6 +78,60 @@ impl Default for UiScale {
     }
 }
 
+/// General representation of progress between two values.
+#[derive(Debug, Clone)]
+pub struct Progress<T>
+where
+    T: Send + Sync + Clone + 'static,
+{
+    /// The minimum value that the progress can have, inclusive.
+    pub min: T,
+    /// The maximum value that the progress can have, inlucsive.
+    pub max: T,
+    /// The current value of progress.
+    pub value: T,
+}
+
+impl<T: Send + Sync + Copy> Progress<T>
```

Comment:
You require `T` to implement `Copy` but in the declaration you require `Clone` instead

Reply:
Good catch. They should be one or the other, not both I think.